src/app-logic/constants.js

```diff
@@ -54,3 +54,38 @@ export const TRACK_VISUAL_PROGRESS_LINE_WIDTH = 2;
 
 // Height of the active tab resources panel header.
 export const ACTIVE_TAB_TIMELINE_RESOURCES_HEADER_HEIGHT = 20;
+
+// ═════════════════════════════════════════════════════════════════════════════
+// Storage and server-related constants
+// ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+// For the 2 values GOOGLE_STORAGE_BUCKET and PROFILER_SERVER_ORIGIN, several
+// values are possible, so that you can easily switch between existing server
+// (both local or remote).
+//
+// GOOGLE_STORAGE_BUCKET
+// ⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺
+// This defines which bucket we fetch profile data at load time.
+
+// Google storage bucket, where production profile data is stored:
+export const GOOGLE_STORAGE_BUCKET = 'profile-store';
```

Comment:
I wondered if I should do something more complex than these constants, for example arrays, with one constant only to change to switch both values. Or using webpack's DefinePlugin and work with environment variables.
In the end I thought this solution is the simplest and especially straightforward, and we can as well move to more complex solutions if we think they're needed. What do you think?

Reply:
I'd keep the simplest thing for now, until we feel the pain of tweaking the values.